### ISSUE TITLE
通道合并是不是反了

### ISSUE THREAD

**Issue by TNTWEN** [bubbliiiing/yolov4-tiny-pytorch#4]
https://github.com/bubbliiiing/yolov4-tiny-pytorch/blob/53851c5bffa044ce9f0f8864791ed45e20a935d8/nets/yolo4_tiny.py#L77

P4 = torch.cat([feat1,P5_Upsample],axis=1)     ->  P4 = torch.cat([P5_Upsample,feat1],axis=1)

**Comment by bubbliiiing** [comment]
我看了看AlexeyAB的CFG，是贴反了，不过对于训练而言影响不大，只是无法使用AlexeyAB的权重

**Comment by TNTWEN** [comment]
是的，我参考了您的网络结构使用AlexeyAB的权重，做OpenVINO的模型适配，发现精度很低，昨天发现是这个通道合并的问题。
带训练的话确实影响不大。

**Comment by bubbliiiing** [comment]
感谢，我懒得改了，因为还得重新训练权重

**Comment by TNTWEN** [comment]
好滴好滴
